<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>my_dialog</class>
 <widget class="QDialog" name="my_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>my_dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="comboBox"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Calculate</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableView">
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>100</number>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="tableView_2">
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>100</number>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="tableView_3">
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>85</number>
     </attribute>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
